src/routes/etudiants/[id]/+page.ts

```diff
@@ -6,11 +6,20 @@ export const load = (async ({ params }) => {
 	const { data: profile, error: err } = await supabase
 		.from('Profile')
 		.select()
-		.eq('user_id', params.id)
+		.eq('id', params.id)
 		.single();
+
 	if (err) {
+		if (err.code === 'PGRST116') {
```

Comment:
`'PGRST116'` ?

Is it a Supabase error code. If so, it might be relevant to put it in a `const` in a util file to use it everywhere.

Reply:
It's a postgres error code but yes i'll create a constant file to handle these codes